pkgs/applications/audio/zrythm/default.nix

```
@@ -161,13 +192,17 @@ stdenv.mkDerivation rec {
     gappsWrapperArgs+=(
       --prefix GSETTINGS_SCHEMA_DIR : "$out/share/gsettings-schemas/${pname}-${version}/glib-2.0/schemas/"
       --prefix XDG_DATA_DIRS : "$XDG_ICON_DIRS"
```

```suggestion
      --prefix XDG_DATA_DIRS : "$XDG_ICON_DIRS:${breeze-icons}/share"
```
we can combine this to generate less code